packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -137,7 +137,7 @@ describe("publish", () => {
 
 			expect(std.out).toMatchInlineSnapshot(`
 			"Attempting to login via OAuth...
-			Opening a link in your default browser: https://dash.cloudflare.com/oauth2/auth?response_type=code&client_id=54d11594-84e4-41aa-b438-e81b8fa78ee7&redirect_uri=http%3A%2F%2Flocalhost%3A8976%2Foauth%2Fcallback&scope=account%3Aread%20user%3Aread%20workers%3Awrite%20workers_kv%3Awrite%20workers_routes%3Awrite%20workers_scripts%3Awrite%20workers_tail%3Aread%20d1%3Awrite%20pages%3Awrite%20zone%3Aread%20offline_access&state=MOCK_STATE_PARAM&code_challenge=MOCK_CODE_CHALLENGE&code_challenge_method=S256
+			Opening a link in your default browser: https://dash.cloudflare.com/oauth2/auth?response_type=code&client_id=54d11594-84e4-41aa-b438-e81b8fa78ee7&redirect_uri=http%3A%2F%2Flocalhost%3A8976%2Foauth%2Fcallback&scope=account%3Aread%20user%3Aread%20workers%3Awrite%20workers_kv%3Awrite%20workers_routes%3Awrite%20workers_scripts%3Awrite%20workers_tail%3Aread%20d1%3Awrite%20pages%3Awrite%20zone%3Aread%20oauth_account_ssl_and_certificates_write%20offline_access&state=MOCK_STATE_PARAM&code_challenge=MOCK_CODE_CHALLENGE&code_challenge_method=S256
```

Comment:
Will this change require users to re-authenticate before they can use the `mtls-certificate` feature? I assume so since new permission is needed.

If so, will we gracefully handle this scenario or what will the experience be like for someone with an existing token that now tries to upload a certificate?

Reply:
Yes, unfortunately. even adding the permission to an existing scope would require re-authenticating as i'm told.

It would be simple enough to drop a try/catch around these methods and display a message like
```
Error: Unauthorized
You may need to log out and log back in
```

I could also detect when the user is using an OAuth token, check to see if it includes the required scope, and if not, prompt for a log out + back in

Reply:
🤔 I'll have to look into this more.

Reply:
@IgorMinar by "central" I think you mean that we catch an authorisation error in the `cfetch()` calls and print a suitable message similar to what Matt suggests above?

Reply:
Yes. If we detect that the request has failed because the token is expired or doesn't have sufficient privileges, could we error gracefully with a actionable error message or even go as far as re-initiate the auth workflow?

Reply:
What I meant by "central" was that this should be a Wrangler feature and not something that mTLS has to implement specifically for their command. Apologies about the ambiguity.

Reply:
I will talk to Josh about how it kicks back an error, then we can look into a Core feature that can potentially do this "even go as far as re-initiate the auth workflow"